notebooks/book1/02/centralLimitDemo.ipynb

```diff
@@ -0,0 +1,343 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    latexify(width_scale_factor=2, fig_height=2)
</pre><p>please set <code>fig_height=1.85</code> I checked in latex source code </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Has been updated. Will submit a PR now